Woof/Woof/View/OwnerFlow/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -20,15 +20,44 @@ struct DetailPetSitterView: View {
                 Spacer()
                 PriceLabel(price: viewModel.price)
             }
+
             Divider()
             Text(viewModel.bio)
                 .font(.system(size: AppStyle.FontStyle.footnote.size))
             Divider()
-            Spacer()
+                .padding(.bottom)
+
+            HStack {
+                Text(slotHeaderLabel)
+                Spacer()
+                DatePicker("",
+                           selection: $viewModel.selectedDate,
+                           in: Date.now...,
+                           displayedComponents: .date).labelsHidden()
+                    .font(.system(size: AppStyle.FontStyle.heading.size))
```

Comment:
nit: This looks more readable:
```suggestion
                           displayedComponents: .date)
                           .labelsHidden()
                           .font(.system(size: AppStyle.FontStyle.heading.size))
```

Reply:
Fixed.